CatalogService.API/Security/AuthExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Cryptography;
+
+namespace CatalogService.API.Security;
+
+public static class AuthExtensions
+{
+    public static IServiceCollection AddAuthSecurity(this IServiceCollection services, IConfiguration configuration)
+    {
+        services.AddAuthentication( options =>
+        {
+            options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+            options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
+        }).AddJwtBearer( options =>
+        {
+            var rsa = RSA.Create();
+            rsa.ImportSubjectPublicKeyInfo(

```

Comment:
What's the idea behind this?
In typical case you generate `JWT` signing keys on IMS side and provide those with `/.well-known/openid-configuration` in OIDC case. Storing signing keys in config would require constant checks if keys are ok.